presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkInjectorFactory.java

```diff
@@ -44,13 +45,27 @@
     private final Map<String, String> configProperties;
     private final Map<String, Map<String, String>> catalogProperties;
     private final List<Module> additionalModules;
+    private final Optional<Module> accessControlModuleOverride;
 
-    public PrestoSparkInjectorFactory(Map<String, String> configProperties, Map<String, Map<String, String>> catalogProperties, List<Module> additionalModules)
+    public PrestoSparkInjectorFactory(
+            Map<String, String> configProperties,
+            Map<String, Map<String, String>> catalogProperties,
+            List<Module> additionalModules)
+    {
+        this(configProperties, catalogProperties, additionalModules, Optional.empty());
+    }
+
+    public PrestoSparkInjectorFactory(
+            Map<String, String> configProperties,
+            Map<String, Map<String, String>> catalogProperties,
+            List<Module> additionalModules,
+            Optional<Module> accessControlModuleOverride)
```

Comment:
I'm not particularly sure if we should have two separate constructors, but I don't have strong opinion on this.

Reply:
Just wanted to keep the main non testing call site clean. Having an extra optional might be confusing.